product_variant_parent_link/__manifest__.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "product_variant_parent_link",
+
+    'summary': """
+        links from product variant to product
+    """,
+
+    'description': """
+        links from product variant to product
+    """,
+
+    'author': "Vertel AB",
+    'website': "http://www.yourcompany.com",
+
+    # Categories can be used to filter modules in modules listing
+    # Check https://github.com/odoo/odoo/blob/14.0/odoo/addons/base/data/ir_module_category_data.xml
+    # for the full list
+    'category': 'Uncategorized',
+    'version': '0.001',
```

Comment:
Please use version standard: OdooMajor.OdooMinor.VersionMajor.VersionMinor.VersionMicro
Eg. 14.0.0.0.1
Also add a short description what is added in this release